pkg/vm/vm.go

```diff
@@ -275,6 +275,14 @@ func (v *VM) AddBreakPoint(n int) {
 	ctx.sc.breakPoints = append(ctx.sc.breakPoints, n)
 }
 
+// RemoveBreakPoint removes the breakpoint in the current context.
+func (v *VM) RemoveBreakPoint(n int) {
```

Comment:
BTW, can you also add a support of respective VM CLI command? Similar to https://github.com/nspcc-dev/neo-go/blob/66fbcb2f0032ae26350bf5b07231419e2195c8d5/cli/vm/cli.go#L104

If not, then let's move it to a separate issue.

Reply:
should I also add a `list breakpoints`?

Reply:
To me it's a useful feature. It may be supported as a separate command and may be even as an extension of `(v *VM) PrintOps` (some special symbol may be added to every output line that has a breakpoint).

But keep in mind that `step` command is also handled with the help of breakpoints:
https://github.com/nspcc-dev/neo-go/blob/66fbcb2f0032ae26350bf5b07231419e2195c8d5/cli/vm/cli.go#L1191

Hence, the output of `list breakpoints` contains not only user-defined-breakpoints. This fact should be mentioned in the command doc.

Reply:
Another note is that we now actually have a set of breakpoint-related commands, hence we may group them under a single `break` command as a set of subcommands, something like: `break add`, `break remove` and `break list`:
```
var commands = []*cli.Command{
	{
		Name: "break",
		Subcommands: []*cli.Command{
			{
				Name: "break",
				...
			},
			{
				Name: "remove",
				...
			},
			{
				Name: "list",
				...
			},
		},
	},
```
But it's an incompatible change. @roman-khimov, what do you think?

Reply:
I think `man gdb`.

Reply:
Well, I had to be more clear: what do you think about *incompatibility* of this change, can we afford this?

Regarding `gdb`:  it has the same concept as proposed above: `gdb breakpoints` has a subset of breakpoint-related commands including `break`, `clear`, `delete`. Although I haven't find something similar to `list` to display all breakpoints.

Reply:
`ib` lists breakpoints in gdb

Reply:
> incompatibility of this change

If you really want to debug something with this CLI you won't be happy typing "breakpoints break" and alike, these:
```
break, brea, bre, br, b -- Set breakpoint at specified location.
delete, del, d -- Delete all or some breakpoints.
```
are all available at the top level in GDB. Also, good commands start with verbs.

Strict compatibility is not critical for this CLI (and can be done with some transition period), but likely we can extend it in a compatible way now.

Reply:
OK, let's then agree on compatible extension and use `delete` and `ib` for breakpoints removal and listing correspondingly.